deploy/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -532,6 +532,33 @@ rules:
   - apiGroups: ["replication.storage.openshift.io"]
     resources: ["volumereplicationclasses/status"]
     verbs: ["get"]
+  - apiGroups: ["csiaddons.openshift.io"]
```

Comment:
The sidecar does not need all these permissions. It only creates the CSIAddonsNode, nothing more. The CSIAddonsNode is created with an ownerRefrence set to the (parent) Pod, so it will be garbage collected automatically when the Pod is deleted.

The following should be sufficient:
```yaml
  - apiGroups: ["csiaddons.openshift.io"]
    resources: ["csiaddonsnodes"]
    verbs: ["create"]
```
None of the other permissions in the `csiaddons.openshift.io` api-group below are needed.